backend/municipal-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -0,0 +1,112 @@
+        facility_selection_file: UploadFile = File(description="Excel file containing facilities"),
+        facility_output: str = Form(default="Facility Output Template",
+                                    description="Name of the sheet containing facility data"),
+        facility_service: FacilityTemplateService = Depends(),
+        request_info: str = Form(default="")
+):
+    input_temp_file = None
+    request_info = request_info_from_json(request_info)
+    get_authorized_request_info(request_info)
+
+    try:
+        input_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)